batch.go

```diff
@@ -28,7 +28,6 @@ func (qq *QueuedQuery) Query(fn func(rows Rows) error) {
 		if err != nil {
 			return err
 		}
-		rows.Close()
```

Comment:
This is not useless. There is no guarantee that fn() will have read all rows. In this case the `rows.Close()` is necessary for the rows to be closed before the error is read in `return rows.Err()`.